src/lib/redux/slices/pitwallSlice/models.ts

```diff
@@ -1,7 +1,13 @@
+import { NO_TIME } from "@/components/utils/constants";
+import {
+  convertMsToDisplay,
+  convertToDelta,
+} from "@/components/utils/formatter/TimeConversion";
+
 export interface PitwallState {
   session: PitwallSession | null;
   gameSession: GameSession | null;
-  liveTiming: LiveTiming[];
+  liveTiming: LiveTimingDto[];
```

Comment:
Shouldn't this still be `LiveTiming[]`?

Reply:
pushed a change to fix it, lmk if you had something else in mind.

Reply:
I changed this on purpose, as I received a lot of warnings when storing a class (non-serializable) / values in the state. Maybe there is a better way we can do it in the future, but for now, I would go with storing the Dto in the state and then return the class from the selector.

Reply:
Oh interesting, okay no prob. I don't have much experience with typescript but i figured it would be possible. Oh well, a selector works 👍